### PR TITLE
1122 update bds finding vignette

### LINKED ISSUE #1122: Documentation:  Update BDS Finding Vignette to include new functions
### Please select a category the issue is focused on?

User Guides

### Let us know where something needs a refresh or put your idea here!

Add new functions created as part of the ADLB workflow to the BDS finding vignette.

### PR DESCRIPTION
**Pull request by yangc71**
Close #1122 

Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/) 
- [x] Updated relevant unit tests or have written new unit tests - See [Unit Test Guide](https://pharmaverse.github.io/admiral/articles/unit_test_guidance.html#writing-unit-tests-in-admiral-)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiral/articles/programming_strategy.html#deprecation-1)?
- [x] Update to all relevant roxygen headers and examples 
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md`
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue so that it closes after successful merging. 
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done!  Much love to your accomplishment!

### CODE REVIEW

**Comment by yangc71** [comment]
Hi @millerg23 @bms63 @konstantinakoukR, I added the new functions we created to the BDS vignette. For the `derive_param_wbc_abs`, I used a small dummy example just to show how the function works, instead of using the LB domain and writing additional codes to do processing (since the vignette is mainly generating ADVS, and ADEG/ADLB were only mentioned as small examples). Please let me know if another approach is preferred.

File: vignettes/bds_finding.Rmd
Comment (bms63) [inline vignettes/bds_finding.Rmd:351]: We should change the heading to this section to - see above

Comment (bms63) [reply vignettes/bds_finding.Rmd:351]: Nevermind, can't make suggestion in code if it hasnt been modified.   Was thinking - 
Derive Additional Parameters (BSA, BMI, MAP for ADVS, QTCF for ADEG and WBC_ABS for ADLB) - not sure if people will know WBC_ABS...

File: vignettes/bds_finding.Rmd
Comment (bms63) [inline vignettes/bds_finding.Rmd:689]:
```suggestion
This function derives a character shift variable concatenating shift in values based on a
```

**Review by bms63 — APPROVED** [review]
Looks great to me!

**Comment by bms63** [comment]
Oh I put a little blurb for the news in the #998 PR for this update to BDS Findings

**Comment by yangc71** [comment]
@bms63 Thanks! But I noticed the Roxygen check failed even after I merged the latest devel. Do you have any suggestions on how to proceed?

**Comment by yangc71** [comment]
> 
Ok thanks, should we wait for others to review or should we merge this PR?

File: vignettes/bds_finding.Rmd
Comment (bms63) [reply vignettes/bds_finding.Rmd:351]: @yangc71 What do you think about this new title?  After that I am happy to merge into devel

Comment (yangc71) [reply vignettes/bds_finding.Rmd:351]: Sorry I misunderstood this comment previously and thought you meant it can be ignored. The current title says **e.g.** BSA, BMI etc., so I thought it's ok to keep as it is since it's just giving examples of new parameters, so we don't have to list out everything? Also WBC_ABS is not standard term so can be confusing to include in the title.

Comment (bms63) [reply vignettes/bds_finding.Rmd:351]: Sounds good!  Merging in!

**Review by bms63 — APPROVED** [review]

**Pull request merged**
